### ISSUE TITLE
pre-trained model available?

### ISSUE THREAD

**Issue by bhomass** [theamrzaki/text_summurization_abstractive_methods#7]
Hi I am interested in models 4 and 5. From past trials it takes many days to train the models from CNN/Daily News data. Plus you may have to re-train to try new parameters. Is it possible to just use your pre-trained model and run some test articles (no abstracts)?

**Comment by theamrzaki** [comment]
here is the pretrained model for [model 4 ](https://drive.google.com/open?id=1lowJjysNn0-vQbO7ymZaVbhmsQ6BMD_R)
you can copy it directly to your google drive if you need to continue training it even further ([more about this](https://hackernoon.com/begin-your-deep-learning-project-for-free-free-gpu-processing-free-storage-free-easy-upload-b4dba18abebc))

This model is pointer generator model ([more about pointer generator model](http://bit.ly/2EhcRIZ)) , however

**Comment by theamrzaki** [comment]
i think if you use google colab for your training , as they offer a gpu for training , you can actually train a model just like this in a couple of sessions (where each session takes about 2 hours and then google colab disconnects , you can simple reconnect it again and continue on with your training) 

So i think it won't take much time (not days) merely couple of sessions to have a model just like this

**Comment by theamrzaki** [comment]
I hope this is helpful :D

**Comment by bhomass** [comment]
I copied over the pre-trained model, but when running the code, it fails to load the checkpoint

I0616 01:52:02.131697 139755304531840 saver.py:1280] Restoring parameters from /content/gdrive/My Drive/CNN/logs/myexperiment/train/model.ckpt-11428
I0616 01:52:02.749844 139755304531840 <ipython-input-8-69ad81a2d1b9>:26] Failed to load checkpoint from /content/gdrive/My Drive/CNN/logs/myexperiment/train. Sleeping for 10 secs...

The code definitely saw the directory and the file. And the dependency classes are loaded based on the requirement.txt file. Why would it fail to load?

**Comment by theamrzaki** [comment]
[model 5 policy gradient model ](https://drive.google.com/open?id=1CFyhfRUi1MulwpwikOPV0cK6r_21Lwlf)
hope it works fine , it must share the very same hyper-parameters in order to work fine

**Comment by theamrzaki** [comment]
[Model 5 Scheduled sampling pretrained model](https://drive.google.com/open?id=1wGfjUlhJi5LCW7bagwyiEdi_4Db8TY3n)
hope this helps

**Comment by ellivalla** [comment]
Hi! Thank you for your tutorials on the Google Colab and Pointer Generator :)

I am trying to run the model 4 code using Google Colab. I have uploaded the data ja myexperiment files to Google Drive, connected Google Drive to Google Colab notebook (succesfully I hope). 

It is currently executing the code, but it is taking a while. I am wondering if it's normal and if there is something I missed. 

Thank you in advance for any kind of guidance.

**Comment by step8p** [comment]
@ellivalla have you changed the runtime accelerator (on colab) to GPU?

**Comment by theamrzaki** [comment]
yes i have chise the gpu option witgin google colab. it makes running much faster
